### PR TITLE
Blank Canvas Blocks: Add default styles for Gallery block

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

The grid is currently broken on the demo site because of an issue with the removal of spaces after CSS minification. This PR handles the rest of the styles needed for this block

#### Related issue(s):

Part of #3413

### CODE REVIEW

**Comment by jffng** [comment]
This is what I'm seeing on BCB, compared to theam demo:

BCB This PR | Blank Canvas theam demo
------- | ------- 
Screen Shot 2021-03-10 at 9 22 42 AM | Screen Shot 2021-03-10 at 9 23 32 AM

Is it because of this: 

> The grid is currently broken on the demo site because of an issue with the removal of spaces after CSS minification.

File: blank-canvas-blocks/sass/blocks/_gallery.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	margin: var(--wp--custom--gallery--margin);
+
+	.blocks-gallery-image,
+	.blocks-gallery-item {
+
+		figcaption {
+			margin: var(--wp--custom--gallery--caption--margin);
+			// Text color is always white to account for default gradient background
+			color: var(--wp--custom--gallery--caption--color);
+			font: var(--wp--custom--gallery--caption--font-size);
```
Comment (jffng) [inline blank-canvas-blocks/sass/blocks/_gallery.scss]: Should this be `font-size` or are you intentionally using the shorthand? If so, I believe font family also needs to be supplied in the value.

**Comment by MaggieCabrera** [comment]
> Is it because of this:
> 
> > The grid is currently broken on the demo site because of an issue with the removal of spaces after CSS minification.

It's not the same issue, but I don't know what's causing it! I thought it was a local issue because when I swapped my site to use classic Blank Canvas I saw the same thing. Inspecting the css didn't give me any clues at all, it all seemed to look the same as the demo site does!

**Comment by jffng** [comment]
The markup is not the same for me locally versus theam demo:

BCB local | theam demo
------ | -------
Screen Shot 2021-03-10 at 9 42 23 AM | Screen Shot 2021-03-10 at 9 42 39 AM

Locally (running Gutenberg master) the gallery is wrapped in a figure 😕

**Comment by MaggieCabrera** [comment]
> The markup is not the same for me locally versus theam demo:


I've been breaking my head and I can't find what changed that broke the gallery but I did confirm that this is an issue on Gutenberg, not on the theme. If you deactivate your GB plugin on your local install, the gallery looks like it should.

**Review by scruffian — APPROVED** [review]

**Pull request merged**